### PR TITLE
Fix warning in screenlock, handle image not found better

### PR DESCRIPTION
**Pull request by gmacd**
Handle case where image isn't found, and tidy code a bit.

Signed-off-by: Graham MacDonald <grahamamacdonald@gmail.com>

### CODE REVIEW

File: sys/src/cmd/screenlock.c [hunk 1]
```diff
@@ -211,62 +211,63 @@ void
...
+	if(initdraw(nil, nil, "screenlock") < 0)
 		sysfatal("initdraw failed");
 	if(display == nil)
 		error("no display");
 
 	/* screen is now open and covered.  grab mouse and hold on tight */
 	procrfork(grabmouse, nil, 4096, RFFDG);
 	procrfork(blanker, nil, 4096, RFFDG);
+
+	draw(screen, screen->r, display->black, nil, ZP);
```
Comment (rminnich) [inline sys/src/cmd/screenlock.c]: You could swap these two lines and do draw(screen, r, etc. etc.)
Not a big deal but ...

**Comment by gmacd** [comment]
Why not?  I need the git practice :)

**Comment by elbing** [comment]
Ok @gmacd, can you merge it or no permissions?

**Comment by gmacd** [comment]
I don't think I have permission

**Pull request merged**